### PR TITLE
Update listMeeting and different view panels for DG

### LINKED ISSUE #175: Update DG with listMeeting implementation

### PR DESCRIPTION
**Pull request by sikai00**
Closes #175 and #179

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -154,6 +154,40 @@ Classes used by multiple components are in the `seedu.address.commons` package.
...
+
+### Different view panels
+
+The GUI changes view panels depending on the last executed command. For example, a `listMeeting` will cause the meeting list view panel to be displayed, while `viewClient i/1` will cause a detailed client view panel to be displayed.
+
+#### Implementation
+
+Below is a sequence diagram that illustrates the execution of `listMeeting` command and the interaction with `Model`.
+
+![DifferentViewPanelsSequenceDiagram](images/DifferentViewPanelsSequenceDiagram.png)
```
Comment (rexong) [inline docs/DeveloperGuide.md:181]: Was thinking if the model is necessary in this context.

**Review by rexong — APPROVED** [review]
Just one point that would like your comments on it. But no big issues that I have spotted. Will approve this iteration.

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (sikai00) [reply docs/DeveloperGuide.md:181]: Yea was considering it as well, though I think I might have to consult TA for this. But thanks for pointing out!

Comment (sikai00) [reply docs/DeveloperGuide.md:181]: Will message him and change accordingly to his comments

**Pull request merged**